### PR TITLE
Portfolio development

### PR DESCRIPTION
**Pull request by Nama-Salameh**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+    flex-direction: column;
+}
+.main__section1__h1 , .main__section1__p , .main__section1--star1{
+    display: block;
+    font-size: 80px;
+    margin: 0;
+    color:transparent;
+    background-image: linear-gradient(to right, #fff, #d8ceced3 , #ffffffbe, #09090989 , #ffffff9f ,   #fff);
+    -webkit-background-clip: text;
+    animation: header-animation 12s linear infinite;
```
Comment (wafa200099) [inline style.css]: please try to make it faster by increasing the time

File: index.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                        </p>
+
+                        <a href="https://github.com/Farha-Dahman/WassilniApp_Project_/tree/Dev" target="_blank"> 
+                            <i class="fab fa-github  github-icon" ></i>
+                        </a>
+
+                        <!--
+                             <a href="https://www.figma.com/file/7D8aeBK74QCf5elBPkIhPW/Wassilni?type=design&node-id=0%3A1&mode=design&t=lEK3N9uXoPPL9Tud-1" target="_blank"> 
+                            <i class="fab fa-figma  figma-icon" ></i>
+                            </a>
```
Comment (hafsakhanfar) [inline index.html]: we always try to  clear our code from comments

File: index.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
```
Comment (wafa200099) [inline index.html:1]: could you please format the code by ctl+ shift + F by prettier  extension

File: index.html [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+                    <div class="About-me-arrow About-me-arrow-first"></div>
+                    <div class="About-me-arrow About-me-arrow-second"></div>
+                </a>
+            </section>
+
+            <section id="Aboutme" class="portfolio__section  portfolio__section2" >
+
+                <img class="main__section2__img" src="images/NamaPhoto.jpeg" alt="Nama' Salameh Photo" title="Nama' Salameh personal photo" height="400px" width="300px" loading="lazy">
+
+                <p class="main__section2__p"> Hi, I am Nama'. Computer Science Apprenticeship Program student at An-Najah National University majoring in Software engineering. My GPA is 3.43 . I have knowledge in HTML, CSS and JavaScript,  which I used in developing web projects for previous sessions. In addition, I have knowledge of MySQL, Firebase and Mongo
```
Comment (wafa200099) [inline index.html]: could you please justify this paragraph content

File: index.html [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1Second.jfif" alt="web project1 image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
+                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1Third.jfif" alt="web project1 image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
+                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1Fourth.jfif" alt="web project1 image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
+                    </div>
+                    <div>
+                        <h3 class="main__section4--project__title"> Rock Scissors paper game </h3>
+                        <p class="main__section4--project__p">A simple game with the computer, taken <br> from the idea of the rock, Paper, <br> Scissors game. We want to make an <br> idea similar to it with the computer <br> in a fun and entertaining way.
```
Comment (wafa200099) [inline index.html]: could you please justify this paragraph content as well

File: index.html [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+                <div class="portfolio__section main__section4--project">
+                    <div class="main__section4--imagecontainer" >
+                        <img class="main__section4--project__img main__section4--project2__img" src="images/project2First.jfif" alt="web project2 image" title="Talabatak website store image" width="350" height="350" loading="lazy">
+                        <img class="main__section4--project__img main__section4--project2__img" src="images/project2Second.jfif" alt="web project2 image" title="Talabatak website store image" width="350" height="350" loading="lazy">
+                        <img class="main__section4--project__img main__section4--project2__img" src="images/project2Third.jpeg" alt="web project2 image" title="Talabatak website store image" width="350" height="350" loading="lazy">
+                    </div>
+                    <div>
+                        <h3 class="main__section4--project__title"> Talabatak -All in one- </h3>
+                        <p class="main__section4--project__p">The Website displays a virtual products <br> store that contains virtual products and <br> contact information.
```
Comment (wafa200099) [inline index.html]: could you please justify this paragraph content as well

File: index.html [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+                        <img class="main__section4--project__img  main__section4--project3__img" src="images/prject3First.jpeg" alt="mobile application project2 image" title="Wassilni application image" width="350" height="350" loading="lazy">
+                        <img class="main__section4--project__img  main__section4--project3__img" src="images/project3Second.jpeg" alt="mobile application project2 image" title="Wassilni application image" width="350" height="350" loading="lazy">
+                        <img class="main__section4--project__img  main__section4--project3__img" src="images/Project3Third.jpeg" alt="mobile application project2 image" title="Wassilni application image" width="350" height="350" loading="lazy">
+                    </div>
+                    <div>
+                        <h3 class="main__section4--project__title"> Wassilni </h3>
+
+                        <p class="main__section4--project__p">A carpooling app helps to provide transportation <br> for people whose places of work are far <br> away. Instead of using public transportation,<br> the application allows them to find a car at <br> a minimal price and in a shorter time.
```
Comment (wafa200099) [inline index.html]: could you please justify this paragraph content as well

File: index.html [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+                    </ul>
+                </div>
+            </section>
+
+            <section id="Projects" class="portfolio__section portfolio__section4">
+                <h2 class="main__section4__h"> My Projects </h2>
+                <div class="portfolio__section main__section4--project">
+                    <div class="main__section4--imagecontainer">
+                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1First.jfif" alt="web project1 image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
```
Comment (wafa200099) [inline index.html]: very nice nama'a all **alt** provided also **loading**  what a good  use

File: index.html [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+                    <li> <a href="#Projects" class="header__link"> Projects </a> </li>
+                    <li> <a href="Nama' Salameh Resume.pdf" class="header__link" target="_blank"> Resume </a> </li>
+                    <li> <a href="#Contact" class="header__link"> Contact </a> </li>
+                </ul>
+            </nav>
+        </header>
+        <main class="portfolio__main">
+
+            <section id="Home" class="portfolio__section  portfolio__section1">
+                <span class="fa fa-star main__section1--star1"></span>
```
Comment (hafsakhanfar) [inline index.html]: it's just a suggestion if you want you can add this star on the other sections in the project with some animation(like move it horizontally with shadow) to it

File: index.html [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+                </div>
+            </section>
+
+            <section id="Projects" class="portfolio__section portfolio__section4">
+                <h2 class="main__section4__h"> My Projects </h2>
+                <div class="portfolio__section main__section4--project">
+                    <div class="main__section4--imagecontainer">
+                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1First.jfif" alt="web project1 image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
+                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1Second.jfif" alt="web project1 image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
+                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1Third.jfif" alt="web project1 image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
```
Comment (wafa200099) [inline index.html]: you can change the alt 

```suggestion
                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1Third.jfif" alt="web project1 main image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
```

File: style.css (hunk 1 above)
Comment (Nama-Salameh) [reply style.css]: Can check it now, I hope it's good.

File: index.html (hunk 9 above)
Comment (Nama-Salameh) [reply index.html]: You can test it if it's good.

File: index.html (hunk 8 above)
Comment (Nama-Salameh) [reply index.html]: I'm really happy for this comment.

**Pull request merged**